packages/types/src/global-settings.ts

```diff
@@ -42,6 +42,10 @@ export const globalSettingsSchema = z.object({
 	customInstructions: z.string().optional(),
 	taskHistory: z.array(historyItemSchema).optional(),
 
+	// Image generation settings (experimental) - flattened for simplicity
+	openRouterImageApiKey: z.string().optional(),
+	imageGenerationSelectedModel: z.string().optional(),
```

Comment:
Should this be openRouterImageGenerationSelectedModel?

Reply:
Yeah I'm just not sure the right structure to support multiple providers. If we don't care about preserving the key when you switch between them, we might want `imageApiProvider`, `imageApiKey`, `imageApiModelId`?